lib/util/ts_parser.js

```diff
@@ -356,8 +359,7 @@ shaka.util.TsParser = class {
 
     while (i < len) {
       const value = data[i++];
-      // optimization. state 0 and 1 are the predominant case. let's handle
-      // them outside of the switch/case
+      // optimization. state 0 and 1 are the predominant case.
       if (state == 0) {
```

Comment:
Please give names to the states as an enum for readability.

Reply:
@joeyparrish I can't give you clear names, but I have put some comments to make it more explanatory. I hope that helps to understand the code better.